app/components/pages/createPoll/index.js

```diff
@@ -3,73 +3,59 @@ import { Formik, Form, FieldArray } from 'formik';
 import Box from '@material-ui/core/Box';
 import Typography from '@material-ui/core/Typography';
 import Button from '@material-ui/core/Button';
-import * as Yup from 'yup';
+import { string, object, date, ref } from 'yup';
 import ClearIcon from '@material-ui/icons/Clear';
 import AddIcon from '@material-ui/icons/Add';
 import PersonIcon from '@material-ui/icons/Person';
 import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
 import SaveIcon from '@material-ui/icons/Save';
 import CustomDatePicker from './datePicker';
 import { Input } from '../../index';
-import { useStyles } from './styles';
 import WrapInBreadcrumbs from '../../layout/wrapInBreadcrumbs/index';
 import WrapInCard from '../../layout/wrapInCard';
 import { H5 } from '../../typography';
 import { Toast } from '../../../utils/helper';
 
-const CreatePollPage = ({
-  heading,
-  inputLabelName,
-  inputLabelQuestion,
-  buttonName,
-  optionButtonName,
-}) => {
-  const classes = useStyles();
+const CreatePollPage = () => {
   const upperLimitForOptions = 4;
   return (
     <WrapInBreadcrumbs>
       <WrapInCard>
         <Formik
           initialValues={{
             options: [''],
-            textFieldName: '',
-            textFieldQuestion: '',
-            startDate: undefined,
-            endDate: undefined,
+            Name: '',
+            Question: '',
+            startDate: new Date(),
+            endDate: new Date(),
           }}
           onSubmit={() => {
             Toast({
               icon: 'success',
               title: `Poll created successfully`,
             });
           }}
-          validationSchema={Yup.object().shape({
-            textFieldName: Yup.string().required('Required'),
-            textFieldQuestion: Yup.string().required('Required'),
-            startDate: Yup.date()
-              .required('Required')
-              .default(() => new Date()),
-            endDate: Yup.date()
-              .required('Required')
-              .when(
-                'startDate',
-                (startDate, schema) =>
-                  startDate &&
-                  schema.min(startDate, 'Must be greater than start date')
+          validationSchema={object().shape({
```

Comment:
define schema at top of the file after imports see create event for reference

Reply:
done